e2e/snapshot-creators/default.cy.snap.js

```diff
@@ -86,10 +86,13 @@ describe("snapshots", () => {
 
   function updateSettings() {
     cy.request("PUT", "/api/setting/enable-public-sharing", { value: true });
-    cy.request("PUT", "/api/setting/enable-embedding", { value: true });
-    cy.request("PUT", "/api/setting/embedding-secret-key", {
-      value: METABASE_SECRET_KEY,
-    });
+    cy.request("PUT", "/api/setting/enable-embedding", { value: true }).then(
```

Comment:
I wanted to make sure we don't run into race conditions between the two calls, making the tests flaky